src/pudl/transform/gridpathratoolkit.py

```diff
@@ -0,0 +1,161 @@
+"""Transformations of the GridPath RA Toolkit renewable generation profiles.
+
+Wind and solar profiles are extracted separately, but concatenated into a single table
+in this module, as they have exactly the same structure. The generator aggregation group
+association tables for various technology types are also concatenated together.
+
+Note that this transform is a bit unusual, in that it is producing a highly processed
+output table. That's because we're working backwards from an archived finished product
+to be able to provide a minimum viable product. If it goes well we will integrate or
+reimplement the steps leading up to this output table later.
+"""
+
+import pandas as pd
+from dagster import AssetCheckResult, AssetIn, asset, asset_check
+
+import pudl
+
+
+def _transform_capacity_factors(
+    capacity_factors: pd.DataFrame, utc_offset: pd.Timedelta
+) -> pd.DataFrame:
+    """Basic transformations that can be applied to many profiles.
+
+    - Construct a datetime column and adjust it to be in UTC.
+    - Reshape the table from wide to tidy format.
+    - Name columns appropriately.
+    """
+    # Convert timestamp to UTC before stacking!
+    # TODO: Ensure that we're using the same hour starting vs. ending convention
+    # here as we use in the FERC-714 and EPA CEMS data. Should this be reflected in
+    # the column names here and elsewhere, whatever the convention is?
+    datetime_cols = ["year", "month", "day", "hour"]
+    capacity_factors["datetime_utc"] = (
+        pd.DatetimeIndex(pd.to_datetime(capacity_factors.loc[:, datetime_cols]))
+        - utc_offset
+    )
+    capacity_factors = (
+        capacity_factors.drop(columns=datetime_cols)
+        .set_index("datetime_utc")
+        .stack()
+        .reset_index()
+    )
+    capacity_factors.columns = ["datetime_utc", "aggregation_group", "capacity_factor"]
+    capacity_factors = capacity_factors.astype({"aggregation_group": "string"})
+    return capacity_factors
+
+
+@asset(io_manager_key="pudl_io_manager")
+def out_gridpathratoolkit__hourly_available_capacity_factor(
```

Comment:
can you add a note in here about why this is an `out_` asset even though it is in the transform module?

Reply:
Sure, I'll move it down here from the module docstring.